.github/workflows/ci.yml

```diff
@@ -47,6 +47,8 @@ jobs:
     runs-on: ubuntu-20.04
     container: golang:1.18-alpine3.16
     timeout-minutes: 20
+    env:
+      INTEGRATION_TESTS: 1
```

Comment:
I tested it manually, and it passes it to the ubuntu machine.

Reply:
Maybe the harness script can set this environment variable? 
I don't see a reason to execute the tests w/o needing this retry

Reply:
What do you mean?
It's just enabling retry with 404 for the integration tests...

I'm probably missing something... or misunderstanding the comment.

Reply:
Your current solution forces the harness executor to configure the environment variable in order to have those retries. 

I'm just saying - why not having the harness test itself configure this environment variable?

Reply:
I found it cleaner than setting an environment variable in the harness.go code.
Since harness calls (by command execution) terraform, I was also concerned the environment variable would not pass through.
Not sure what's the scope of the environment when setting in GO and then running a terraform command.

(In other words it's me being a little more cautious). Happy to test it out if you believe it's preferred.

Reply:
We're executing the harness tests also internally in env0. My motivation was to avoid needing to set the environment variable on our end. 

I think it's preferred to have it (or any other indicator that's different than an environment variable) defined in the `harness.go` file so it'd indicate that code is running as a part of the harness/integration tests

Reply:
BUT (forgot to say) if you find it too complex then let's leave it as is

Reply:
What I could test is setting the environment variable in harness.go

E.g.: from harness.go
```
	err = os.Setenv("TF_CLI_CONFIG_FILE", path.Join(cwd, "tests", "terraform.rc"))
	if err != nil {
		log.Fatalln("Unable to set env:", err)
	}```

I think this will solve the concern you have.

Reply:
@yaronya - modified. Setting environment variable each time terraform command is called.

Reply:
Great, thanks
I'll approve but just making sure you validated that the environment variable is available to the `terraform` bin execution 🙏